### ISSUE TITLE
Missing files for 12th gen and newer

### ISSUE THREAD

**Issue by mikolaj612** [intel/Intel-Linux-Processor-Microcode-Data-Files#80]
Hello, as thread name suggest, files for 12th gen and newer CPU's are missing in your newest package, despite it is mentioned in the release notes.

An oversight probably?

**Comment by whpenner** [comment]
Yes.  There are links to specification updates for products that are not being updated.  We noticed that late in the process of posting and chose not to remove them.  The table of Updated Platforms is correct as is the table of Security updates.

**Comment by mikolaj612** [comment]
> Yes. There are links to specification updates for products that are not being updated. We noticed that late in the process of posting and chose not to remove them. The table of Updated Platforms is correct as is the table of Security updates.

Any ETA for missing microcode files?

**Comment by whpenner** [comment]
The microcode update files are not missing.  There are no updates for those files this cycle.  The previous files are the latest files available.

**Comment by xry111** [comment]
But https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01046.html says 12 and 13 gens are affected.  So is it fixed for 12 and 13 gens with an earlier release, or for 12 and 13 gens a BIOS update is needed?

**Comment by whpenner** [comment]
Ah, now I see the issue.  Thanks for pointing this out.  Intel-SA-01046 issue was mitigated in the MCUs that were released in May (what we call IPU 24.2) and this release is referenced in the above linked document.  Due to the timing of public disclosure, we made the changes in the previous MCU and are now disclosing the issue.  I do realize that this isn't obvious and we will work to make this more clear in the future.

**Comment by hmh** [comment]
Did the same thing happen for CVE-2024-24980 / Intel-SA-01100, and signature 0xc06f2 ?  Was it fixed by 20230514, but disclosed now along with the updates for the other affected processors?

https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01100.html

**Comment by whpenner** [comment]
@hmh: Yes.
